res/skins/Deere/style.qss

```diff
@@ -175,6 +175,10 @@ QTableView {
   gridline-color: red;
 }
 
+QTableView:focus, QTreeView:focus {
+  border: 1px solid #027ED7;
```

Comment:
We have have different focus colors for search bar, and tableview/treeview.
Since only one of the widgets can have focus, should we settle for one color, like with the other skins?

Reply:
Right, so let's go with blue which fits the color scheme nicely and double the border width, or keep orange. Any preference? @Be-ing

Reply:
It seems we mixed something up: search already has a blue border and it's like this for quite a while now. So my doubts the contrast would be too low does not apply, and we didn't receive complaints.

Reply:
My point was, that in Deere we still have ``#FF6600`` for search line focus https://github.com/mixxxdj/mixxx/blob/6d13a9885a7ae0ce06e321166ba6e804041c1af2/res/skins/Deere/style.qss#L275-L281, while the treeview and library table focus is blue.

So we need to either change the search line focus to blue too, or apply the search line focus color to  treeview and library table focus.

Blue is not the best focus for imo, since all other active states are already blue, but i´m ok with whatever color you choose.

Reply:
So I mixed something up, sorry.
hm, I thought blue would fit BECAUSE all indicators are blue, but since I have no preference let's make it orange.